НЕ МЕРЖИТЬ: Cинхронный режим загрузчиков тестов

artbear opened the pull request:
Пока не реализованы
- [ ] загрузка вложенных каталогов
- [x] открытие внешних инструментов
- [ ] Не работает вызов файлов/каталогов из истории

EvilBeaver committed:
Асинхронный режим загрузчиков тестов

EvilBeaver committed:
Отключены избыточные проверки

artbear commented:
Пока не реализованы
+ загрузка вложенных каталогов
+ открытие внешних инструментов

artbear commented:
Остальные фичи отлично работают.
Говорим большущее спасибо @EvilBeaver :)

artbear commented:
Данный PR исправляет ошибку https://github.com/xDrivenDevelopment/xUnitFor1C/issues/646

PR закрывает #521

artbear commented:
Не работает вызов файлов/каталогов из истории

artbear commented:
Ура, также заработали
+сохранение открываемых файлов/каталог в истории 
+автооткрытие последнего файла/каталога при открытии браузера тестов

EvilBeaver commented:
@artbear - 

* сохранение открываемых файлов/каталог в истории
* автооткрытие последнего файла/каталога при открытии браузера тестов

У меня не заработали. Я что-то пропустил?

artbear committed:
ОФ+УФ: Добавлено открытие внешних инструментов в режиме отказа от модальности и синхронности

artbear committed:
ОФ+УФ: Добавлено исходники кода открытия внешних инструментов в режиме отказа от модальности и синхронности

EvilBeaver committed:
Merge pull request #1 from artbear/feature/v4-async

ОФ+УФ: Добавлено открытие внешних инструментов в режиме отказа от модальности и синхронности

EvilBeaver committed:
реализован асинхронный прогон в пакетном режиме

EvilBeaver commented:
@artbear https://github.com/EvilBeaver/xUnitFor1C/commit/2aaee8c6cf97bfa921fc237551b16321bd24fd00
Я не знаю, как правильно ты мержишь пулреквесты с конфликтами. Вот коммит, что с ним делать?

EvilBeaver committed:
По сравнению с синхронным вариантом сломалось поведение, если тестов не найдено.

1dimir committed:
Исправлена опечатка в имени переменной

ПараметрЗапуска - свойство глобального контекста, Строка
ПараметрыЗапуска - структура из параметров процедуры ВыполнитьПакетныйЗапускСинхронно

artbear commented:
Я смержил изменения в отдельной ветке.
Проверю ее и волью в девелоп.

EvilBeaver committed:
Исправлена ошибка запуска в синхронном режиме

EvilBeaver committed:
Merge pull request #2 from 1dimir/patch-1

Исправлена опечатка в имени переменной
